.gitignore

```diff
@@ -10,3 +10,5 @@
 
 /dist
 /node_modules
+
+/test/node_smoke_tests/lib/ensure_iterability.js
```

Comment:
I could put it all in `dist` and not add anything to `.gitignore` or `.jshintignore` but then I'd have to copy the entire `test/node_smoke_tests` directory over as the `ensure_iterability_es6.js` file requires files relative to its original location so transpiling to a different directory wouldn't fly.